### ISSUE TITLE
Vue 3 support / migration?  

### ISSUE THREAD

**Issue by havok2063** [spacetelescope/jdaviz#211]
jdaviz is currently using Vue 2.x but Vue 3.0 is soon to be released. It's currently in Release Candidate but planned for full release in August.  Looks like Vuetify is planning to switch over in Q3/Q4 2020.  I imagine this will facilitate an update for `ipyvuetify` as well.  I imagine the August release date might be too close to the September build deadline to make the switch, especially if Vuetify hasn't updated yet.   It looks like Vue3 has some significant improvements over 2.x, so I support migrating, but it also has a fair amount of breaking API changes.  Maybe we should start planning a roadmap for switching over.   

Vue3 changes and migration guide - https://v3.vuejs.org/guide/migration/introduction.html#overview
Vuetify Timeline - https://vuetifyjs.com/en/introduction/roadmap/#the-vuetify-roadmap

I suspect MAST will want to move to Vue3 as soon as possible, so we should coordinate, and potentially explore any backwards compatibility options.

**Comment by nmearl** [comment]
Indeed, we've been keeping our eye on the next version, but I'm confident this won't be considered until after the initial delivery of the tools.

I'd also like to point to the more fine-grained development overview for the Vuetify transtion:

- [Their kanab board](https://www.notion.so/d107077314ca4d2896f0eeba49fe8a14?v=5cc7c08e9cc44021a7c86a20f189b0ba) with development tasks.
- [Their (currently empty) overview](https://www.notion.so/Breaking-Changes-c00f6f98523e449da7d628cac0093235) for the breaking changes in the transition.

**Comment by havok2063** [comment]
Ok, great!  Thanks @nmearl.  No breaking changes yet but that's a long list of no-status tasks.
